src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -503,4 +503,7 @@
   <data name="ResourceStreamEndedUnexpectedly" xml:space="preserve">
     <value>Resource stream ended at {0} bytes, expected {1} bytes.</value>
   </data>
+  <data name="SharedArgumentMissing" xml:space="preserve">
+    <value>Argument required after '/shared:'. </value>
```

Comment:
It's required because the `:` was specified.  It is legal to have `/shared` or `/shared:somevalue` but not `/shared:`.  

I'm up for any better wording here.